nav2_msgs/action/ComputePathToPose.action

```diff
@@ -1,14 +1,14 @@
 # Error codes
 # Note: The expected priority order of the errors should match the message order
-int16 NONE=0
-int16 UNKNOWN=1
-int16 TF_ERROR=2
-int16 START_OUTSIDE_MAP=3
-int16 GOAL_OUTSIDE_MAP=4
-int16 START_OCCUPIED=5
-int16 GOAL_OCCUPIED=6
-int16 TIMEOUT=7
-int16 NO_VALID_PATH=8
+int16 NONE=1000
```

Comment:
Lets give ourselves room to grow: start the controller server at 1000 and then offset the rest accordingly. For all we know, something serious could be added between the controller and the base in the future (right now we actually already have a couple: velocity smoother + collision monitor - though neither are in the nav2 bt framework)